lib/future-selectors.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+var futureEvents = require('../data/next-months.json')
+var eventSelectors = require('./event-selectors');
+
+module.exports = function () {
+  return futureEvents.map(eventSelectors)
```

Comment:
I think you want a .reverse() here so that it shows the nearest events at the top of the list.